internal/cmd/web.go

```diff
@@ -61,8 +62,7 @@ and it takes care of all the other things for you`,
 
 // checkVersion checks if binary matches the version of templates files.
 func checkVersion() {
```

Comment:
Since we're now embedding all these files, I think it's time to get rid of this function entirely!